server/db/repository/WishCardRepository.js

```diff
@@ -163,6 +163,14 @@ async function getWishCardByAgencyId(agencyId) {
   }
 }
 
+async function getNotDonatedCards() {
+  try {
+    return WishCard.find({ isDonated: false, status: 'published' }).exec();
```

Comment:
isDonated has been removed, so you only have to use status: 'published' 👍 

Edit: Since isDonated is removed, you can use getWishCardsByStatus instead. No need to add another method :)